.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           command: |
             bash scripts/install_third_party.sh
             python scripts/third_party_size_check.py
-            python scripts/pre_commit_linter.py --path=. --verbose
+            python -m scripts.pre_commit_linter --path=. --verbose
```

Comment:
Just wondering ... what is the rationale for this change, and should we be doing it for other scripts as well?

If not, it might be worth adding a comment here to clarify why this line is treated specially.

Reply:
I changed this as now all the python scripts would behave as modules and the pre_commit_linter needed to import `utils.py` and that is only possible if we treat it as a module. This is also true for other python scripts which will be created later but currently only this python file is called directly from the terminal

Reply:
OK, please add a comment explaining this.

Reply:
Done

Reply:
You need to do the same for Travis as well.

Reply:
(Please ignore the above comment. Thanks!)

Reply:
Let me confirm, do I need to add a comment for Travis? Thanks

Reply:
Oh, do add a comment for Travis too. Thanks!

Reply:
Done